src/Plugin.php

```diff
@@ -129,6 +133,28 @@ public static function admin_enqueue_scripts() {
     wp_enqueue_style('showrooms-admin-css', static::getBaseUrl() . '/dist/styles/showrooms-admin.css');
   }
 
+  /**
+   * @implements enqueue_scripts.
+   */
+  public static function wp_enqueue_scripts() {
+    if (!is_product_category()) {
+      return;
+    }
+
+    $query = get_queried_object();
+    if (empty($query->slug)) {
```

Comment:
WP requires all terms to have a slug.  Which special case is covered by this condition?  This should be documented in an inline comment.

Reply:
This check looks unnecessary.